src/app/containers/reference-form-dialog/reference-contributors-list.tsx

```diff
@@ -0,0 +1,194 @@
+  }, [setAnchorEl]);
+
+  const handleToggleRefContributorType = useCallback(() => {
+    if (isGroupRefContributor(contributor)) {
+      onChange({ firstName: '', lastName: contributor['groupName'] });
+    } else {
+      onChange({ groupName: `${contributor['lastName']} ${contributor['firstName']}` });
+    }
+    closeMenu();
+  }, [contributor, onChange, closeMenu]);
+
+  const handleDelete = useCallback(() => {
+    onDelete();
+    closeMenu();
+  }, [closeMenu, onDelete]);
+
+  const classes = useReferenceContributorStyles();
+
+  return (
+    <section className={classes.contributorInputFields}>
+      <DragHandle />
+      {isGroupRefContributor(contributor)
+        ? renderGroupForm(classes, contributor, handleFormChange)
+        : renderRefContributorForm(classes, contributor, handleFormChange)}
+      <IconButton onClick={openMenu} classes={{ root: classes.menuButton }}>
+        <MoreVertIcon fontSize="small" />
+      </IconButton>
+      <Menu anchorEl={anchorEl} keepMounted open={Boolean(anchorEl)} onClose={closeMenu}>
+        <MenuItem test-id={'toggle-contributor-type-btn'} onClick={handleToggleRefContributorType}>
+          {isGroupRefContributor(contributor) ? `Set to individual ${entityName}` : `Set to group ${entityName}`}
+        </MenuItem>
+        <MenuItem test-id={'delete-contributor-btn'} onClick={handleDelete}>
+          Delete {entityName}
+        </MenuItem>
+      </Menu>
+    </section>
+  );
+});
+
+export const ReferenceContributorsList: React.FC<ReferenceRefContributorsListProps> = ({
+  addCtaLabel,
+  className,
+  entityName,
+  refContributors,
+  onChange
+}) => {
+  const handleRefContributorChange = useCallback(
+    (index: number) => (refContributor: ReferenceContributor) => {
+      const updatedRefContributorsList = [...refContributors];
+      updatedRefContributorsList[index] = refContributor;
+      onChange(updatedRefContributorsList);
+    },
+    [onChange, refContributors]
+  );
+
+  const addNewRefContributor = useCallback(() => {
+    onChange([...refContributors, { firstName: '', lastName: '' }]);
+  }, [onChange, refContributors]);
+
+  const handleRefContributorDelete = useCallback(
+    (index: number) => () => {
+      const updatedRefContributorList = [...refContributors];
+      updatedRefContributorList.splice(index, 1);
+      onChange(updatedRefContributorList);
+    },
+    [refContributors, onChange]
+  );
+
+  const onSortEnd = useCallback(
+    ({ oldIndex, newIndex }) => {
+      const refContributorsList = [...refContributors];
+
+      const movedRefContributor = refContributorsList[oldIndex];
+      refContributorsList.splice(oldIndex, 1);
+      refContributorsList.splice(newIndex, 0, movedRefContributor);
+      onChange(refContributorsList);
+    },
+    [onChange, refContributors]
+  );
+
+  return (
+    <SortableList onSortEnd={onSortEnd} axis="y" className={className} useDragHandle={true}>
+      {refContributors.map((contributor, index) => (
```

Comment:
One for Nick, but in places where there is both an Authors and Editors list it might be useful to add a label before the list so that it is clear what the list is.